Project.toml

```diff
@@ -41,8 +41,8 @@ GPUArraysCore = "0.1"
 JSON = "0.21"
 KernelAbstractions = "0.9"
 MuladdMacro = "0.2"
-PointNeighbors = "0.4.2"
-Polyester = "0.7.5"
+PointNeighbors = "0.4.5"
```

Comment:
0.4.2 is enough. We don't need to require 0.4.5. No new features of 0.4.5 are used in TrixiP.

Reply:
Should we put 0.4 than instead?

Reply:
No, we get unsatisfiable compat requirements with 0.4.1 and TrixiParticles. We need at least 0.4.2.